src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExtensionMethodImportCompletionProviderTests.cs

```diff
@@ -1888,6 +1888,63 @@ public static void Bar(this Goo goo, int x)
             }
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task TestCommitWithSemicolonForMethod()
+        {
+            var markup = @"
+public class C
+{
+}
+namespace AA
+{
+    public static class Ext
+    {
+        public static int ToInt(this C c)
+            => 1;
+    }
+}
+
+namespace BB
+{
+    public class B
+    {
+        public void M()
+        {
+            var c = new C();
+            c.$$
+        }
+    }
+}";
+
+            var expected = @"
+using AA;
+
+public class C
+{
+}
+namespace AA
+{
+    public static class Ext
+    {
+        public static int ToInt(this C c)
+            => 1;
+    }
+}
+
+namespace BB
+{
+    public class B
+    {
+        public void M()
+        {
+            var c = new C();
+            c.ToInt();
+        }
+    }
+}";
+            await VerifyProviderCommitAsync(markup, "ToInt", expected, commitChar: ';', sourceCodeKind: SourceCodeKind.Regular);
+        }
```

Comment:
Add a test for non-invocation context

Reply:
After some thoughts, I feel for ExtensionMethodImportCompletionProvider, it is OK not to check if  the context is delegate or not.
Because 
```C#
public static class Extension
{ 
    public static int ToInt(this Bar b) => 1;
}
```
It is illegal to do this
` Func<Bar, int> fun = bar.ToInt;`
Therefore I feel it is OK to always add parenthesis if the commit char is ';'